api/s2n.h

```diff
@@ -110,6 +112,15 @@ extern int s2n_config_set_extension_data(struct s2n_config *config, s2n_tls_exte
 extern int s2n_config_send_max_fragment_length(struct s2n_config *config, s2n_max_frag_len mfl_code);
 extern int s2n_config_accept_max_fragment_length(struct s2n_config *config);
 
+extern int s2n_config_set_session_state_lifetime(struct s2n_config *config, uint64_t lifetime_in_secs);
+
+extern int s2n_config_set_session_tickets_onoff(struct s2n_config *config, uint8_t enabled);
```

Comment:
my 2 cents, I like having separate functions for enable and disable that way it's easy to know what is happening without looking at the function arguments.